### PR TITLE
Update docs to be consistent with Elastic docs

### PR DESCRIPTION
**Pull request by brianmcgue**
See https://www.elastic.co/guide/en/app-search/current/authentication.html#authentication-signed.

The docs pretty clearly state you need to use a private key with read access.

This caused confusion in https://github.com/elastic/sdh-enterprise-search/issues/636.

### CODE REVIEW

File: README.md
Comment (constancecchen) [inline README.md:373]:
```suggestion
Signed search keys keep your read-only Private API Keys secret and restrict what a user can search over. They can be used to restrict top-level search options. The example below creates a signed search key that will only return the `title` field.

See https://www.elastic.co/guide/en/app-search/current/authentication.html#authentication-signed for more options and documentation.
```

Comment (constancecchen) [reply README.md:373]: ^ updated the copy to match the Elastic docs a bit more closely (helping note what signed search keys do) as well as links to the docs for more options

**Review by constancecchen — APPROVED** [review]
LGTM! Not sure if you want to wait for Chris on this to merge though

**Review by JasonStoltz — CHANGES_REQUESTED** [review]
Wait, what? You don't have to use a private key, you can use a search key.

File: README.md
Comment (richkuz) [inline README.md:373]:
```suggestion
Signed search keys keep your read-only private API keys secret and restrict what a user can search over. They can be used to restrict top-level search options. The example below creates a signed search key that will only return the `title` field.
```

Just a minor suggestion to change [capitalization of feature names](https://brand.elastic.co/302f66895/p/072ccc-naming-guide). Else, everything LGTM 👍  Thanks for submitting this doc change!

**Review by richkuz — APPROVED** [review]

File: README.md
Comment (constancecchen) [reply README.md:373]: FYI that was copied from the linked Elastic docs, so we should update it there too

**Comment by brianmcgue** [comment]
Thanks, @JasonStoltz for halting this. It does indeed seem to work with a search key. I'm going to close out this PR, though we could open another to change the copy if we wanted.

**Pull request closed without merging**
